nginx/plan.sh

```diff
@@ -1,10 +1,12 @@
 pkg_name=nginx
 pkg_origin=core
-pkg_version=1.8.0
+pkg_version=1.10.1
+pkg_description="NGINX web server."
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('bsd')
-pkg_source=http://nginx.org/download/nginx-1.8.0.tar.gz
-pkg_shasum=23cca1239990c818d8f6da118320c4979aadf5386deda691b1b7c2c96b9df3d5
+pkg_source=http://nginx.org/download/nginx-${pkg_version}.tar.gz
+pkg_upstream_url=http://hg.nginx.org/nginx/
```

Comment:
This is better expressed as http://nginx.org/ - instead of the source, we display the information site.